### ISSUE TITLE
Cannot find module '@material/foo` 

### ISSUE THREAD

**Issue by Prathik-Jain** [trimox/angular-mdc-web#1925]
```
ERROR in node_modules/@angular-mdc/web/checkbox/typings/checkbox.d.ts(7,39): error TS2307: Cannot find module '@material/checkbox'.
node_modules/@angular-mdc/web/chips/typings/chip-set.d.ts(6,38): error TS2307: Cannot find module '@material/chips/chip-set'.
node_modules/@angular-mdc/web/chips/typings/chip.d.ts(6,35): error TS2307: Cannot find module '@material/chips/chip'.
node_modules/@angular-mdc/web/dialog/typings/dialog.component.d.ts(8,37): error TS2307: Cannot find module '@material/dialog'.
node_modules/@angular-mdc/web/drawer/typings/drawer.d.ts(5,74): error TS2307: Cannot find module '@material/drawer'.
node_modules/@angular-mdc/web/floating-label/typings/floating-label.d.ts(3,44): error TS2307: Cannot find module '@material/floating-label'.
node_modules/@angular-mdc/web/form-field/typings/character-counter.d.ts(3,56): error TS2307: Cannot find module '@material/textfield/character-counter'.
node_modules/@angular-mdc/web/icon-button/typings/icon-button.d.ts(6,47): error TS2307: Cannot find module '@material/icon-button'.
node_modules/@angular-mdc/web/line-ripple/typings/line-ripple.d.ts(3,41): error TS2307: Cannot find module '@material/line-ripple'.
node_modules/@angular-mdc/web/linear-progress/typings/linear-progress.d.ts(3,45): error TS2307: Cannot find module '@material/linear-progress'.
node_modules/@angular-mdc/web/list/typings/list.d.ts(6,35): error TS2307: Cannot find module '@material/list'.
node_modules/@angular-mdc/web/menu-surface/typings/menu-surface-base.d.ts(4,42): error TS2307: Cannot find module '@material/menu-surface'.
node_modules/@angular-mdc/web/notched-outline/typings/notched-outline.d.ts(4,45): error TS2307: Cannot find module '@material/notched-outline'.
node_modules/@angular-mdc/web/radio/typings/radio.d.ts(7,36): error TS2307: Cannot find module '@material/radio'.
node_modules/@angular-mdc/web/slider/typings/slider.d.ts(5,37): error TS2307: Cannot find module '@material/slider'.
node_modules/@angular-mdc/web/snackbar/typings/snackbar.component.d.ts(5,39): error TS2307: Cannot find module '@material/snackbar'.
node_modules/@angular-mdc/web/switch/typings/switch.d.ts(6,37): error TS2307: Cannot find module '@material/switch'.
node_modules/@angular-mdc/web/tab-bar/typings/tab-bar.d.ts(7,37): error TS2307: Cannot find module '@material/tab-bar'.
node_modules/@angular-mdc/web/tab-indicator/typings/tab-indicator.d.ts(4,83): error TS2307: Cannot find module '@material/tab-indicator'.
node_modules/@angular-mdc/web/tab-scroller/typings/tab-scroller.d.ts(5,42): error TS2307: Cannot find module '@material/tab-scroller'.
node_modules/@angular-mdc/web/tab/typings/tab.d.ts(5,34): error TS2307: Cannot find module '@material/tab'.
node_modules/@angular-mdc/web/top-app-bar/typings/top-app-bar.d.ts(5,102): error TS2307: Cannot find module '@material/top-app-bar'.
```

**What Angular MDC version are you using?**
1.0.5
**What OS are you using?:**
Linux
###### Related to #1863

**Comment by Prathik-Jain** [comment]
I had to add the `"material-components-web": "^2.3.1"` dependency manually, to get this working.

**Comment by trimox** [comment]
@Prathik-Jain Thanks for reporting. A fix PR was pushed today #1924, and I plan on publishing 2.0.0 today.

**Comment by trimox** [comment]
Closing, please update to 2.0.0.
